pepper-apis/src/main/java/org/broadinstitute/ddp/util/LiquibaseUtil.java

```diff
@@ -41,26 +40,34 @@
 
     public static final String AUTH0_TENANT_MIGRATION = "src/main/resources/db-changes/tenant-migration.xml";
 
-    private Connection conn;
+    private HikariDataSource dataSource;
 
-    private static Connection createLiquibaseConnection(String dbUrl) throws SQLException {
-        DriverManagerConnectionFactory driverManagerConnectionFactory = new DriverManagerConnectionFactory(dbUrl, null);
-        return driverManagerConnectionFactory.createConnection();
+    private LiquibaseUtil(String dbUrl)  {
+        HikariConfig config = new HikariConfig();
+        config.setJdbcUrl(dbUrl);
+        dataSource = new HikariDataSource(config);
+    }
+
+    @Override
+    public void close()  {
+        if (dataSource != null) {
+            dataSource.close();
+        }
     }
 
     /**
      * Runs the global liquibase migrations against the given database url using a connection that's auto-closed.
      */
     public static void runLiquibase(String dbUrl, DB db) {
         try {
-            try (Connection liquibaseConnection = LiquibaseUtil.createLiquibaseConnection(dbUrl)) {
-                LiquibaseUtil liquibaseUtil = new LiquibaseUtil(liquibaseConnection);
+            try (LiquibaseUtil liquibaseUtil = new LiquibaseUtil(dbUrl)) {
```

Comment:
Now that we have hikari, no need to fiddle with lower level driver stuff, so I cleaned this up a bit.